prometheus.go

```diff
@@ -14,20 +14,37 @@ func (g *Gaz) InitPrometheus(path string) {
 	if !strings.HasPrefix(path, "/") {
 		path = "/" + path
 	}
+	g.prometheusRegistry = prometheus.NewRegistry()
 	Sugar.Infof("Setup Prometheus handler at %s", path)
 	g.Router.Handle(path, promhttp.Handler()).Methods("GET")
```

Comment:
How does it work with the promhttp.Handler() ?
Does it know about your gaz.prometheusRegistry?

Reply:
that was a bug, I will create the handler manually to use the gaz registry since this method was using the default global registry